quicklogic/qlf_k4n8/utils/repacker/pb_type.py

```diff
@@ -97,7 +97,9 @@ def yield_pins(self, range_spec=None):
                 if i1 > i0:
                     indices = range(i0, i1 + 1)
                 elif i1 < i0:
-                    indices = range(i0, i1 - 1, -1)
+                    # Do not yield in reverse order when indices are reversed
+                    indices = range(i1, i0 + 1)
+                    # indices = range(i0, i1 - 1, -1)
```

Comment:
Maybe leftover commented code? If it is not required I guess we can remove it

Reply:
Right, fixed.